docroot/themes/custom/mass_theme/mass_theme.theme

```diff
@@ -4377,6 +4377,24 @@ function mass_theme_preprocess_field__person_org_role(&$variables) {
   $variables['person_org_roles'] = $person_roles;
 }
 
+/**
+ * Data to fetch the alerts as soon as possible.
+ *
+ * @see docroot/themes/custom/mass_theme/templates/layout/html.html.twig
+ */
+function mass_theme_build_data_from_alert_blocks(&$variables) {
+  $is_front_page = \Drupal::service('path.matcher')->isFrontPage();
+  /** @var Drupal\Core\Routing\CurrentRouteMatch */
+  $current_route_match_service = Drupal::service('current_route_match');
+  /** @var \Drupal\node\Entity\Node */
+  $node = $is_front_page ?: $current_route_match_service->getParameter('node');
```

Comment:
It looks $node will be true when we are  on the front-page. That’s unexpected to me.

Reply:
I did this because we don't need to load the node if we are on the front page. But I agree, it is confusing. I will update that specific line. Thanks!